src/app/shared/libraries/accordion-item/accordion-item.component.html

```diff
@@ -0,0 +1,15 @@
+
+<div class="govuk-accordion__section">
+  <div class="govuk-accordion__section-header">
+    <h2 class="govuk-accordion__section-heading">
+      <span class="govuk-link govuk-!-font-size-36">{{title}}</span>
+      <span class="govuk-accordion__section-button" id="accordion-default-heading"></span>
```

Comment:
I think it might need to be test-accordion-default-heading if we have an AC for that so that it can be picked up by the automation team.